src/transformers/models/mpnet/configuration_mpnet.py

```diff
@@ -114,3 +117,22 @@ def __init__(
         self.initializer_range = initializer_range
         self.layer_norm_eps = layer_norm_eps
         self.relative_attention_num_buckets = relative_attention_num_buckets
+
+
+class MPNetOnnxConfig(OnnxConfig):
```

Comment:
How about instead of changing the signature of the `forward()` function, we add something like this to the config:

```
torch_onnx_minimum_version = version.parse("1.12") # Pin the appropriate version
```

This way, we avoid introducing a breaking change to the modeling code.